securedrop/client.py

```diff
@@ -167,6 +170,25 @@ async def add_contact(self):
         if msg != "":
             print("Failed to add contact: ", msg)
 
+    async def list_contacts(self):
+        msg = ""
+        try:
+            with open(TEST_FILENAME, 'w') as f:
```

Comment:
This is a reasonable solution for testing, but I don't think a new file should be created when not testing. A bool flag "debug" should probably be passed to `main` (with default `False`) to enable debugging.

Reply:
Changed